test/dlc_tests/container_tests/bin/testDataservice

```diff
@@ -3,6 +3,7 @@ BIN_DIR=${HOME_DIR}/bin
 LOG_DIR=${HOME_DIR}/logs
 
 pip install tf-nightly
+pip install tensorflow==2.3
```

Comment:
Is this running outside the container? If not why are we installing tf-nightly and tensorflow==2.3?